Server/Components/Pages/GamePage.razor

```diff
@@ -54,10 +54,22 @@
 
 @code {
 	private List<Item> equipment = new();
-	private PopupContainer popup;
+	private List<Item> inventory = new();
```

Comment:
nitpick but i think we've decided on private fields having the format `_name`

i really need to start enforcing it more strictly on others

Reply:
It could be added in the .editorconfig so it will be checked automatically and marked as warnings/errors in the VS/VSCode, I will try to add it.